File: test_javaPrj_01/src/programmers/lv0/문자열정렬하기1.py

import re
class 문자열정렬하기1:

    def __init__(self, my_string:str) -> None:

        

        my_string = re.sub('[a-zA-Z]', '', my_string)

        print(list(map(int, sorted( re.sub('[a-zA-Z]', '', my_string)))))
        

        pass

문자열정렬하기1('hi12392')

File: test_javaPrj_01/src/programmers/lv0/문자열정렬하기1.java
package programmers.lv0;

import java.util.Arrays;

public class 문자열정렬하기1 {
    
    private 문자열정렬하기1(){
        try{
            Solution();
            
        }catch(Exception e){
            e.printStackTrace();
        }
    }

    @SuppressWarnings("unused")
    public void Solution() throws Exception{

        String my_string = "hi12392";

        String[] arr = my_string.replaceAll("[a-zA-Z]", "").split("");
        Arrays.sort(arr);

        // return Arrays.stream(arr).mapToInt(obj -> Integer.valueOf(obj)).toArray();
        
        
        // System.out.println(String.join("", arr));

    }


    public static void main(String[] args) {
        new 문자열정렬하기1();
    }


}
